draft-ietf-dnsop-domain-verification-techniques.md

```diff
@@ -145,7 +145,13 @@ If a provider has an application-specific need to have multiple verifications fo
 
 This again allows the provider to query only for application-specific records it needs, while giving flexibility to the user adding the DNS verification record (i.e. they can be given permission to only add records under a specific prefix by the DNS administrator). Whether or not multiple verifying records can exist for the same domain is up to the implementation.
 
-Providers MUST provide clear instructions on when a verifying record can be removed. The user SHOULD de-provision the resource record(s) provisioned for a DNS-based domain verification challenge once the challenge is complete.
+Providers MUST provide clear instructions on when a verifying record can be removed. The user SHOULD de-provision the resource record(s) provisioned for a DNS-based domain verification challenge once the challenge is complete. These instructions MAY be encoded in the RDATA via comma-separated ASCII key-value pairs {{RFC1464}} using the key `expiry`. If this is done, the token should have a key `token`. For example:
```

Comment:
should we say "one time challenge" instead of "challenge" to make it clearer?

Reply:
these instructions MAY be encoded? Don't we want SHOULD be encoded? As in we really want the RDATA to stand on its own

Reply:
Let me take a stab at this.